remoting/tests/src/test/java/org/jboss/as/remoting/RemotingLegacySubsystemTestCase.java

```diff
@@ -178,12 +178,6 @@ public void testSubsystemWithConnectorProperties() throws Exception {
         for (AttributeDefinition ad : RemotingSubsystemRootResource.LEGACY_ATTRIBUTES) {
             assertFalse(ad.getName(), subsystem.hasDefined(ad.getName()));
         }
-        ModelNode endpoint = subsystem.get(RemotingEndpointResource.ENDPOINT_PATH.getKey(), RemotingEndpointResource.ENDPOINT_PATH.getValue());
-        for (AttributeDefinition ad : RemotingEndpointResource.ATTRIBUTES.values()) {
-            ModelNode dflt = ad.getDefaultValue();
-            assertEquals(ad.getName(), dflt == null ? new ModelNode() : dflt, endpoint.require(ad.getName()));
-        }
```

Comment:
Note to self: This needs to be removed because changing to aliases means the values no longer show up in a read-resource.

Reply:
@pferraro is this commit absolutely required for this work?

This Flag.ALIAS addition results in the attributes not appearing in a normal read-resource, which makes the resource basically pointless, as it only exists for compatibility reasons. And it is pointless anyway, and is deprecated. But really we should have removed it in 29 Alpha1 and probably should now (and port that downstream.)

IOW if we must break it we should probably break it all the way and port that downstream. That's a bigger task though. @darranl FYI.

Reply:
> @pferraro is this commit absolutely required for this work?

Yes.  The current /subsystem=remoting/configuration=endpoint resource has 2 fundamental issues:

- The runtime behavior of its attributes do not comply with their management description.
- The worker attribute definition is not consistent between domain vs server wrt capability references - which, once the correct attribute definition flags are applied, otherwise breaks galleon.

WFCORE-6407 ensures that the parameters of the OperationDefinition for the add operation of a given resource are consistent with the parameters used the OperationStepHandler handling the add operation.  Without this commit, add operation parameters as generated by https://github.com/pferraro/wildfly-core/blob/WFCORE-6407/controller/src/main/java/org/jboss/as/controller/SimpleResourceDefinition.java#L245 will not be correct, and lots of model compatibility tests will fail.

> This Flag.ALIAS addition results in the attributes not appearing in a normal read-resource, which makes the resource basically pointless, as it only exists for compatibility reasons.

Presumably, retaining compatibility is not pointless, unless this deprecation is sufficiently old.

> But really we should have removed it in 29 Alpha1 and probably should now (and port that downstream.)

If this child resource can be removed, all the better.